Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -178,7 +192,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
             app.UseRouting();
             app.UseStaticFiles();
 
-            app.UseCors(CorsPolicyName);
+           // app.UseCors(CorsPolicyName);
```

Comment:
Please don't disable CORS, I'm guessing this was done for local testing.  If there is a need for using additional tooling (like web Postman) then we can probably get a ticket open to loosen the CORS restrictions for local development without impacting the release version.